README.md

```diff
@@ -315,124 +303,132 @@ Spec:
 metadata:
   name: kogito-data-index
 spec:
-  # number of pods to be deployed
+  # Number of pods to be deployed
   replicas: 1
-  # image to use for this deploy
+  # Image to use for this deployment
   image: "quay.io/kiegroup/kogito-data-index:latest"
   kafka:
-    # the service name and port for the kafka cluster. Example: my-kafka-cluster:9092
+    # Service name and port for the Kafka cluster, for example, my-kafka-cluster:9092
     serviceURI: my-cluster-kafka-bootstrap:9092
 ```
 
-#### Install Data Index Service with oc client
+#### Installing the Kogito Data Index Service with the oc client
 
-You can use the CR file showed above as a reference and create the custom resource from the command line:
+To create the Kogito Data Index resource using the oc client, you can use the CR file from the previous example as a reference and create the custom resource from the command line as shown in the following example:
 
 ```bash
-# clone this repo
+# Clone this repository
 $ git clone https://github.com/kiegroup/kogito-cloud-operator.git
 $ cd kogito-cloud-operator
-# make your changes
+# Make your changes
 $ vi deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml
-# deploy to the cluster
+# Deploy to the cluster
 $ oc create -f deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml -n my-project
 ```
 
-You should be able to access the GraphQL interface via the route created for you:
+You can access the GraphQL interface through the route that was created for you:
 
 ```bash
 $ oc get routes -l app=kogito-data-index
 
-NAME                HOST/PORT                                                                      PATH   SERVICES            PORT   TERMINATION   WILDCARD
-kogito-data-index   kogito-data-index-kogito.apps.mycluster.example.com                                   kogito-data-index   8080                 None
+NAME                HOST/PORT                                                  PATH   SERVICES            PORT   TERMINATION   WILDCARD
+kogito-data-index   kogito-data-index-kogito.apps.mycluster.example.com               kogito-data-index   8080   None
 ```
 
 ## Kogito CLI
 
-A CLI tool is available to make it easy to deploy new Kogito Services from source instead of relying on CRs yaml files.
+The Kogito CLI tool enables you to deploy new Kogito services from source instead of relying on CRs and YAML files.
 
-### CLI Requirements
+### Kogito CLI requirements
 
-1. [`oc` client](https://docs.okd.io/latest/cli_reference/get_started_cli.html) installed
-2. An authenticated OpenShift user with permissions to create resources in a given namespace
-3. Kogito Operator [installed](#installation) in the cluster
+- The [`oc` client](https://docs.okd.io/latest/cli_reference/get_started_cli.html) is installed.
+- You are an authenticated OpenShift user with permissions to create resources in a given namespace.
+- The [Kogito Operator](#kogito-operator-installation) is installed in the cluster.
```

Comment:
I don't see it as a requirement as the CLI tries it install it on project where it is not (or at least the CLI will ask to install it via OperatorHub).

Reply:
Removed